### PR TITLE
Adds some info comments to CoreBundle Configuration

### PR DESCRIPTION
**Pull request by NiklasBr**
I'm certain that, at a minimum, my description of `status_cache` is wrong here, but I don't fully understand it after reading the code. So it would be nice to have some short documentation here.

### CODE REVIEW

File: bundles/CoreBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -472,6 +472,7 @@ private function addAssetNode(ArrayNodeDefinition $rootNode)
                         ->addDefaultsIfNotSet()
                         ->children()
                             ->integerNode('max_pixels')
+                                ->info("Maximum number of pixels an image can have when added (width × height).")
                                 ->defaultValue(40000000)
```
Comment (jdreesen) [inline bundles/CoreBundle/DependencyInjection/Configuration.php]: Maybe we can improve the formatting of the number a bit. I, at least, have trouble to tell how much it is without the underscores.

```suggestion
                                ->defaultValue(40_000_000)
```

File: bundles/CoreBundle/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -561,9 +566,11 @@ private function addAssetNode(ArrayNodeDefinition $rootNode)
                                         ])
                                     ->end()
                                     ->booleanNode('status_cache')
+                                        ->info("Store image metadata in a cache")
```
Comment (mattamon) [inline bundles/CoreBundle/DependencyInjection/Configuration.php]: As far as I checked this is more about a general performance improvement.
https://github.com/pimcore/pimcore/pull/11702 

As brusch mentions in his first comment:
This improves the performance significantly when using (remote) object storage for website projects.

Does this enhance your understanding on the config?

**Review by mattamon — CHANGES_REQUESTED** [review]

File: bundles/CoreBundle/DependencyInjection/Configuration.php (hunk 2 above)
Comment (NiklasBr) [reply bundles/CoreBundle/DependencyInjection/Configuration.php]: Think I got a better grasp of it, thanks @mattamon.

**Review by mattamon — APPROVED** [review]
LGTM, descriptions are helpful!

**Pull request merged**